docs/visualization/path/response_time.md

```diff
@@ -10,9 +10,9 @@ Please refer to [FAQ](../../faq/faq.md#how-response-time-is-calculated) if you a
     However, both will be referred to as "response time" for simplicity in this document.
 <prettier-ignore-end>
 
-Three cases of response time is defined; `best-to-worst`, `best`, and `worst`. With `best` case, CARET samples shortest time elapsed in a targeted path. With `worst` case, the previous message input timing is taken into account. `best-to-worst` case includes most of all cases between `best` and `worst`.
+Three cases of response time is defined; `all`, `best`, `worst`, and `worst-with-external-latency`. With `all` case, CARET samples all time elapsed in a targeted path. With `best` case, CARET samples shortest time elapsed in a targeted path. With `worst` case, CARET samples largest time elapsed in a targeted path. With `worst-with-external-latency` case, the previous message input timing is taken into account. `best-to-worst` case includes most of all cases between `best` and `worst`.
```

Comment:
```suggestion
As a visualisation of the graph, four different cases can be specified in the `case` argument. Default value is 'all'; 'all', 'best', 'worst', and 'worst-with-external-latency'.

- `all` case
  - CARET displays the elapsed time from all input times in the same cycle.
- `best` case
  - CARET displays the elapsed time that is the shortest of all input times in the same cycle.
- `worst` case
  - CARET displays the elapsed time that is the longest of all input times in the same cycle. 
- 'worst-with-external-latency` case
  - CARET displays the elapsed time from the last input time of a previous cycle.
```

Reply:
Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).